packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -178,231 +182,231 @@ describe('Overhead controller', () => {
   })
   describe('full feature', () => {
```

Comment:
Why not `describe.skip`? With the test body commented out the test output sort of suggests that the test runs / the condition is covered.

Reply:
This doesn't exist in tap.

Reply:
You _can_ do `describe(name, fn, { skip: true })`, though that may not be great on the end of a large, multi-line describe. 🤔

Reply:
Fixed.